test/unit/chain.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @module test/unit/chain
+ */
+
+'use strict';
+
+require('chai').should();
+
+const Account = require('core/account');
+const chain = require('core/chain');
+
+const SECRET_KEYS = [
+    Buffer.from('557dce58018cf502a32b9b7723024805399350d006a4f71c3b9f489f7085cb50', 'hex'),
```

Comment:
What is the reason for hard-coded secret keys? If no secret key is provided, random secret key will be generated on each account creation (see [core/account.js](https://github.com/WingsDao/griffin-consensus-poc/blob/delegates/core/account.js#L34-L36)).